.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
+
+  matrix:
+    - ATOM_CHANNEL=stable
+    - ATOM_CHANNEL=beta
+
+language: ruby
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+os:
```

Comment:
I had issues getting rvm to work with OS X on travis, e.g. https://travis-ci.org/jschroeder9000/linter-haml/jobs/115572141.